docs/V4_API_Updates.md

```diff
@@ -177,21 +238,25 @@ to your project, and `#import RevenueCat-Swift.h` in your bridging header. You c
 		</tr>
 	</thead>
 	<tbody>
+		<tr>
+			<td>invalidatePurchaserInfoCache</td>
+			<td>invalidateCustomerInfoCache</td>
+		</tr>
 		<tr>
 			<td>logIn(_ appUserId, _ completion)</td>
-			<td>login(appUserId:completionBlock)</td>
+			<td>login(appUserId:completion)</td>
```

Comment:
this is actually `logIn`, with capital `I`

Reply:
✅